u3-views/app.js

```diff
@@ -0,0 +1,34 @@
+const express = require('express')
+const expHbs = require('express-handlebars')
+const {index} = require('./features/indexController')
+const { courseDetails,courseUnits,courseStudents } = require('./features/courseController')
+const { videoDetails1,videoDetails2 } = require('./features/videoController')
+
+const app = express()
+
+// Templates
+app.set('views', './views')
+app.set('view engine', 'hbs')
+app.engine('hbs', expHbs({
+  extname: '.hbs',
+  defaultLayout: false,
+  partialsDir: ['./views/partials', './views/layouts']
+}))
+
+
+// Middleware
+app.use(express.static('public'))
+
+// Routes
+app.get('/', index)
+app.get('/courses/:code', courseDetails)
+app.get('/courses/:code/units', courseUnits)
+app.get('/courses/:code/students', courseStudents)
+app.get('/videos/128', videoDetails1)
+app.get('/videos' , videoDetails2)
```

Comment:
เป็นการดีที่คุณใช้ `lowerCamelCase` สำหรับชื่อตัวแปรของคุณ อย่างไรก็ตาม `videoDetails1` และ `videoDetails2` ไม่ได้แสดงถึงความหมายของตัวแปรเหล่านี้ จริง ๆ แล้ว `videoDetails2` คือ controller ที่แสดงรายการวิดีโอ ฉันแนะนำ:

It's good that you are using `lowerCamelCase` for your variable names. However, `videoDetails1` and `videoDetails2` do not represent the meaning on these variables. `videoDetails2` is actually the controller that shows a list of videos; I suggest:
```js
app.get('/videos/128', videoDetails)
app.get('/videos' , videoList)
```